tests/baselines/reference/conditionalTypeContextualTypeSimplificationsSuceeds.types

```diff
@@ -11,8 +11,8 @@ interface Props {
 }
 
 function bad<P extends Props>(
->bad : <P extends Props>(attrs: string extends keyof P ? { [K in keyof P]: P[K]; } : { [K_1 in keyof P]: P[K_1]; }) => void
->    : ^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+>bad : <P extends Props>(attrs: string extends keyof P ? { [K_1 in keyof P]: P[K_1]; } : { [K in keyof P]: P[K]; }) => void
+>    : ^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

Comment:
This is a curious change; how did these end up swapping?

Reply:
Good catch - the `visitNode` calls in `visitExistingNodeTreeSymbolsWorker` in the conditional branch were calling back into `visitExistingNodeTreeSymbolsWorker`, rather than the `visitExistingNodeTreeSymbols` function that handled entering scopes, so we were skipping the mapped type scopes here.